### PR TITLE
Log binning 2d

### PR DESCRIPTION
**Pull request by ricleal**
- There was an error in the vertical axis of the workspace. Fixed that.
- Added the new option to the HFIR command interface help file.
- Some code improvements....

Tests should pass. 

No Release notes. No ticket.
---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- [x] Is the code of an acceptable quality?
- [x] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)? Is it well structured with small focussed classes/methods/functions?
- [x] Are there unit/system tests in place? Are the unit tests small and test the a class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- [x] Do changes function as described? Add comments below that describe the tests performed?
- [ ] How do the changes handle unexpected situations, e.g. bad input?
- [x] Has the relevant documentation been added/updated?
- [ ] Is user-facing documentation written in a user-friendly manner?
- [ ] Has developer documentation been updated if required?

- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Review by mdoucet — APPROVED** [review]
Looks good. :shipit:

File: Framework/Algorithms/src/Qxy.cpp [hunk 1]
```diff
@@ -378,6 +379,21 @@ std::vector<double> Qxy::logBinning(double min, double max, int num) {
   return outBins;
 }
 
+double
+Qxy::getQminFromWs(const API::MatrixWorkspace_const_sptr &inputWorkspace) {
```
Comment (quantumsteve) [inline Framework/Algorithms/src/Qxy.cpp]: If the pointer is never NULL, change the signature to `const API::MatrixWorkspace &`

Comment (ricleal) [inline Framework/Algorithms/src/Qxy.cpp]: Need more more details :) TTYL! Off to the doctor soon.

**Comment by ricleal** [comment]
@quantumsteve Don't forget to merge this :)

**Pull request merged**